### DISCUSSION TITLE
Image Browser: ImageReward is not installed, cannot be used.

### DISCUSSION THREAD

**Discussion by githubUser01946** [vladmandic/sdnext#1374] [Q&A]
I keep seeing that error in CMD when I update the extensions and wondered how to fix it please?

Thanks

**Comment by vladmandic** [comment]
don't need to. that is optional component for imagebrowser. you can install it if you want, but its not an error.

**Reply by githubUser01946** [reply]
Is there anything to gain by installing it?

**Reply by monoinyo** [reply]
It's cool to have another option to sort by an aesthetic rating

**Reply by esmadat** [reply]
In my case I can't see or select anything, not even the extensions bar.
